compile_kaitai_parsers.py

```diff
@@ -14,6 +21,46 @@
 MANIFEST_PATH: Path = KAITAI_PARSERS_DIR / "manifest.json"
 
 
+def find_files_with_excluded_licenses(directory, license_list) -> list[str]:
```

Comment:
Backward compatibility with Python <3.10
```suggestion
def find_files_with_excluded_licenses(directory, license_list) -> List[str]:
```